app/views/merchants/index.html.erb

```diff
@@ -0,0 +1,18 @@
+<html>
+
+  <body>
+    <div class="pageheaders">
+      <h1>All Merchants</h1>
+    </div>
+
+      <div class="links"><%= link_to "New Merchant", "/merchants/new" %></div>
+      <br><br>
+      <div class="merchantsindex">
+        <table>
+          <% @merchants.each do |merchant| %>
+            <tr><th><a style="color: #000000; text-decoration: none;" href= <%= "/merchants/#{merchant.id}"%>> <%= merchant.name %></a></th></tr>
```

Comment:
Same thing here, and there were a few other places that I skipped over... but use link_to when you can.